app/lib/comment/commentDbManager.ts

```diff
@@ -27,6 +27,36 @@ export function getAllCommentsForTweet(tweetId: string) {
     })
 }
 
+export function getLatestComment(tweetId: string) {
+    return GlobalPrisma.comment.findFirst({
+        select: {
+            id: true,
+            text: true,
+            createdAt: true,
+            modifiedAt: true,
+            Owner: {
+                select: {
+                    id: true,
+                    name: true,
+                    displayName: true,
+                    profilePicturePath: true,
+                }
+            },
+            _count: {
+                select: {
+                    likes: true
+                }
+            },
+        },
+        where: {
+            tweetId
+        },
+        orderBy: {
+            createdAt: 'desc'
+        }
+    })
+}
+
 export function createComment(tweetId: string, userId: string, text: string) {
     return GlobalPrisma.comment.create({
```

Comment:
This is what I was referring to, this method already returns the newly created comment. Find where it is used and let that controller return this.